tests/i18n/urls_default_unprefixed.py

```diff
@@ -7,5 +7,6 @@
     re_path(r"^(?P<arg>[\w-]+)-page", lambda request, **arg: HttpResponse(_("Yes"))),
     path("simple/", lambda r: HttpResponse(_("Yes"))),
     re_path(r"^(.+)/(.+)/$", lambda *args: HttpResponse()),
+    re_path(_(r"^users/$"), lambda *args: HttpResponse(), name="users"),
```

Comment:
path() seems more appropriate here ☺️

Reply:
I would need to add the translations of `users/` as here `^users/$` is being translated :cry:  
We don't have/I couldn't find any existing suitable translations of non-regex paths for the i18n tests to update this to something else.
Should I do this update and add translations?